pkg/sysinfo/sysinfo_linux.go

```diff
@@ -302,3 +303,89 @@ func readProcBool(path string) bool {
 	}
 	return strings.TrimSpace(string(val)) == "1"
 }
+
+// defaultMaxCPUs is the normal maximum number of CPUs on Linux.
+const defaultMaxCPUs = 8192
+
+func isCpusetListAvailable(requested, available string) (bool, error) {
+	parsedAvailable, err := parseUintList(available, 0)
+	if err != nil {
+		return false, err
+	}
+	// Start with the normal maximum number of CPUs on Linux, but accept
+	// more if we actually have more CPUs available.
```

Comment:
What is the point of having this `maxCPUs` variable?

Reply:
Let me add that  to the comment